react/week2/todolist/src/TodoList.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import TodoItem from "./TodoItem";
+
+function TodoList({ todos }) {
+  const [itemsArray, setItemsArray] = useState(todos);
+
+  function addNewItem() {
+    const newItem = {
+      id: Math.floor(Math.random() * 100),
+      description: "Random Text",
+    };
+    setItemsArray(itemsArray.concat(newItem));
+  }
+
+  function deleteItem(e) {
+    const idDeleted = e.target.previousSibling.id;
+    setItemsArray(
+      itemsArray.filter(
+        (item) => item.id + item.description.slice(0, 2) !== idDeleted
+      )
+    );
+  }
```

Comment:
Instead of above codes for delete
  const deleteItem = (id) => {
    const todos = [...itemArray];
    setItemsArray(todos.filter((todo) => todo.id !== id));
  };

Reply:
Thanks for the suggestions. I will fix these :)